graph_def_editor/node.py

```diff
@@ -69,6 +83,11 @@ def __init__(self,
       device: TensorFlow device specification string indicating where this node
         should be located. Default value of "" means "use the default device"
     """
+    _type_check(g, graph.Graph, "g")
+    _type_check(node_id, int, "node_id")
+    _type_check(name, str, "name")
```

Comment:
This would be a problem for Python 2 support, but I believe we dropped that a while ago so should be fine.